docs/sources/troubleshoot/support_bundle.md

```diff
@@ -36,6 +36,7 @@ to verify the code used to generate these bundles.
 A support bundle contains the following data:
 * `alloy-components.json` contains information about the [components][components] running on this {{< param "PRODUCT_NAME" >}} instance, generated by the 
 `/api/v0/web/components` endpoint.
+* `alloy-environment.txt` contains the values of several environment variables relevant to the golang runtime.
```

Comment:
Do we need to add these env vars to: https://grafana.com/docs/alloy/latest/reference/cli/environment-variables/

Reply:
There's definitely value to add GOGC, HTTPS_PROXY, GOMAXPROCS documentation. AUTOMEMLIMIT is also worth documenting though it's specific to a module that we use, rather than the go runtime itself.

In reviewing documentation I could find elsewhere GOEXPERIMENT is probably not relevant, looks like that's only used at build time so I'll remove that from this PR.

Reply:
Can we add that to this PR? Or pop up a new PR after this one is complete?

Reply:
Since I updated the docs anyway, I added https://github.com/grafana/alloy/issues/2075 as well.